docs/user/introduction.asciidoc

```diff
@@ -169,9 +169,8 @@ them in bulk operations.
 === Secure {kib}
 
 {kib} offers a range of security features for you to control who has access to what.
-The security features are automatically turned on when
-{ref}/security-minimal-setup.html[security is enabled in
-{es}]. For a description of all available configuration options,
+{ref}/configuring-stack-security.html[Security is enabled automatically] when you enroll {kib} with an {es} cluster.
+For a description of all available configuration options,
```

Comment:
```suggestion
{ref}/configuring-stack-security.html[Security is enabled automatically] when you enroll {kib} with a secured {es} cluster.
For a description of all available configuration options,
```

Reply:
Minor clarification, but otherwise LGTM. I debated getting into how to use the `elasticsearch-create-enrollment-token` tool to generate a token for Kibana, but that's more low-level than should be part of this introduction.